script.js

```diff
@@ -18,3 +18,63 @@ toggleButtonOff.addEventListener('click', () => {
 function closeMenu(toggleButton) {
   navbarLinks.classList.toggle('active')(bodyColor.classList.toggle('change')(toggleScroll.classList.toggle('fixed')));
 }
+
+const toggleTonicPop = document.getElementsByClassName('project-btn')[0];
+const tonicPopup = document.getElementsByClassName('tonic-popup')[0];
+const tonicPopClose = document.getElementsByClassName('toggle-popup')[0];
+const popupBlur = document.getElementsByClassName('header-section')[0];
+const blurNav = document.getElementsByClassName('toolbar')[0];
+
+toggleTonicPop.addEventListener('click', () => {
+  tonicPopup.classList.toggle('show')(popupBlur.classList.toggle('blur')(blurNav.classList.toggle('dim')));
+});
+
+tonicPopClose.addEventListener('click', () => {
+  if (tonicPopup.classList.toggle('show')(popupBlur.classList.toggle('blur')(blurNav.classList.toggle('dim')))) {
+    tonicPopClose();
+  }
+});
+
+const toggleMultiPop = document.getElementsByClassName('project-btn2')[0];
```

Comment:
It was [required](https://github.com/microverseinc/curriculum-html-css/blob/096130a7c3e71e15c3c97ddc2ed0ec6467621a8c/portfolio/8_popup.md?plain=1#L40) to use JavaScript objects to store the information about the projects and load them to the DOM from JavaScript. Could you implement this too?